scripts/plot_whisker.py

```diff
@@ -22,7 +25,14 @@
 commands = [b["command"] for b in results]
 times = [b["times"] for b in results]
 
-plt.boxplot(times, labels=commands)
+boxplot = plt.boxplot(times, vert=True, patch_artist=True)
+cmap = plt.cm.get_cmap("rainbow")
+colors = [cmap(val / len(times)) for val in range(len(times))]
+
+for patch, color in zip(boxplot["boxes"], colors):
+    patch.set_facecolor(color)
+
+plt.title(args.title)
+plt.legend(handles=boxplot["boxes"], labels=commands, loc="best", fontsize="medium")
 plt.ylabel("Time [s]")
-plt.ylim(0, None)
```

Comment:
Please do not remove this. I'm okay with adding a command-line option to specify this somehow, but I believe I would like to keep this the default.

Reply:
Done